Makefile

```diff
@@ -1,5 +1,5 @@
 all: lkmpg.tex
-	pdflatex -shell-escap lkmpg.tex
+	pdflatex -shell-escape lkmpg.tex
```

Comment:
Don't touch `Makefile`. Minimize the functional changes.

Reply:
I've already discarded this change!